integration/util.py

```diff
@@ -134,15 +134,18 @@ def _run_node(reactor, node_dir, request, magic_text):
     # on windows, "tahoe start" means: run forever in the foreground,
     # but on linux it means daemonize. "tahoe run" is consistent
     # between platforms.
```

Comment:
this comment is migrating further an further from the code that picks "run" over "start", maybe it could move down.